osfoffline/application/main.py

```diff
@@ -106,10 +106,9 @@ def start(self):
 
         # todo: HANDLE LOGIN FAILED
         try:
-            user = self.get_current_user()
+            user = session.query(User).one()
         except MultipleResultsFound:
-            debug_trace()
-            self._logout_all_users()
```

Comment:
The original function (application/main.py L165) appears to be unused now, and can be removed to prune dead code.